contracts/src/v0.8/automation/dev/v2_3/AutomationRegistry2_3.sol

```diff
@@ -180,17 +182,19 @@ contract AutomationRegistry2_3 is AutomationRegistryBase2_3, OCR2Abstract, Chain
               fastGasWei: report.fastGasWei,
               linkUSD: report.linkUSD,
               nativeUSD: _getNativeUSD(hotVars),
+              billingToken: billingTokenParams,
               isTransaction: true
             }),
             report.upkeepIds[i]
           );
-          transmitVars.totalPremium += receipt.premium;
-          transmitVars.totalReimbursement += receipt.reimbursement;
+          transmitVars.totalPremium += receipt.premiumJuels;
+          transmitVars.totalReimbursement += receipt.gasReimbursementJuels;
```

Comment:
i like this change, we always pay in link, so Juels are helpful for readability.